src/mobility_canonical_validator/mobility_data.py

```diff
@@ -0,0 +1,50 @@
+import json
+import time
+import requests
+from .constants import Constants
+
+
+class MobilityData:
+    def __init__(self, logger):
+        self.timeout = 5
+        self.max_retries = 10
```

Comment:
@TaskarCenterAtUW/tdei-gs , is 50 secs enough to handle the biggest file we will encounter? Could we check with David or someone from MobilityData on what they suggest as the max time out for the validator?